src/service/xhr-impl.js

```diff
@@ -182,6 +184,26 @@ export class Xhr {
   }
 
   /**
+   * @param {!string} input URL
+   * @param {Array<string>} opt_responseHeaderNames List of response headers to
+   * extract and return.
+   * @param {FetchInitDef} opt_init Fetch options object.
+   * @return {!Promise<!FetchResponse>}
+   */
+  fetch(input, opt_init) {
+    const init = opt_init || {};
+    // The "real" fetch API does not specify the response type in the request.
+    // The fetch response object lets you extract the body in any of the types.
+    // Here, specify 'arraybuffer' in case we use the polyfill, since that is
+    // what we will use when we use fetch.
+    init.responseType = 'arraybuffer';
```

Comment:
The others (`#fetchJson` and `#fetchDocument`) aren't in the non-polyfilled version either. I'd much rather have an explicit `#fetchArrayBuffer` vs having a `#fetch` that defaults to `arraybuffer`.